### ISSUE TITLE
[Bug]: Sync watch status from Jellyfin back to TA not working

### ISSUE THREAD

**Issue by arisenfromtheashes** [tubearchivist/tubearchivist-jf-plugin#32]
### I've read the documentation

- [X] I'm running the latest version of tubearchivist-jf-plugin.
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

unraid docker

## Describe the bug
Failing to sync watch status from Jellyfin back to TA
## Steps To Reproduce
Configured the plugin per the documentation, Library populated properly, marked an episode as watched, didnt sync on its own after some time. Tried to run the scheduled task manually and still fails. (for context i only have the JF to TA config populated and not bidirectional)
## Expected behavior
Should sync the watch status over to TA and be marked with the checkbox

### Relevant Jellyfin log output

```shell
[2024-12-04 11:48:20.069 -06:00] [INF] [172] Jellyfin.Plugin.TubeArchivistMetadata.Plugin: Starting Jellyfin->TubeArchivist playback progresses synchronization.
[2024-12-04 11:48:20.071 -06:00] [ERR] [172] Emby.Server.Implementations.ScheduledTasks.TaskManager: Error executing Scheduled Task
System.ArgumentException: Invalid username (Parameter 'name')
   at Jellyfin.Server.Implementations.Users.UserManager.GetUserByName(String name)
   at Jellyfin.Plugin.TubeArchivistMetadata.Tasks.JFToTubearchivistProgressSyncTask.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.ScheduledTasks.ScheduledTaskWorker.ExecuteInternal(TaskOptions options)
[2024-12-04 11:48:20.071 -06:00] [INF] [172] Emby.Server.Implementations.ScheduledTasks.TaskManager: "JFToTubeArchivistProgressSyncTask" Failed after 0 minute(s) and 0 seconds
```

**Comment by joshrivers** [comment]
Oddly, I get no playhead sync at all...but I do get checkbox on completion. I do not, however, see any logging. (I wish I could provide better details)

**Comment by DarkFighterLuke** [comment]
@arisenfromtheashes from the logs I see that the username you provided in the plugin settings for the JF->TA synchronization is wrong.

**Comment by Jurrer** [comment]
Hi, I realized my issue https://github.com/tubearchivist/tubearchivist-jf-plugin/issues/44 might be related. 
The input fields for the username are not working as they should.
TLDR: use the second field, first one does not work

**Comment by DarkFighterLuke** [comment]
I've just released the version 1.3.7, please try with that and let me know if this bug disappeared.

**Comment by p-rintz** [comment]
This specific issue works for me with the update.

However, the videos dont seem to get auto deleted after being watched.

**Comment by p-rintz** [comment]
Makes sense why Tubearchivist is no longer deleting watched videos then.

Because the plugin updates the 'watched_date' all the time for all videos.

I checked a video that I watched 2 weeks ago or so and the 'watched_date' was set to 6 minutes ago.

Not sure if it also has to do with all videos bein in the "Continue Watching" section after turning on progress syncing.

Im not sure with which criteria TubeArchivist puts videos in that section. I cannot see the position variable set for videos in that section that I never actually started.

**Comment by DarkFighterLuke** [comment]
I am not understanding what the problem is and what is the cause according to you, can you please point me that out?
